packages/graphql/src/translate/translate-read.ts

```diff
@@ -32,14 +32,17 @@ function translateRead({ node, context }: { context: Context; node: Node }): [st
     const optionsInput = resolveTree.args.options as GraphQLOptionsArg;
     const varName = "this";
 
+    const hasOffset = Boolean(optionsInput?.offset) || optionsInput?.offset === 0;
+    const hasLimit = Boolean(optionsInput?.limit) || optionsInput?.limit === 0;
```

Comment:
What does it mean a limit or offset of 0?
I would expect that be the same as no offset or no limit. :thinking:

Reply:
I think this makes sense - if I set the limit to zero then really I would expect zero results back. Without this, setting offset to zero would return all results, which is misleading.